cartographer_rviz/cartographer_rviz/drawable_submap.h

```diff
@@ -70,6 +72,10 @@ class DrawableSubmap : public QObject {
   // 'current_tracking_z'.
   void SetAlpha(double current_tracking_z);
 
+  // todo
```

Comment:
they are not implemented. Remove for now and add back when they are actually added?